MobileWallet/Screens/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,155 @@
+//  ProfileViewController.swift
+
+/*
+	Package MobileWallet
+	Created by Gabriel Lupu on 04/02/2020
+	Using Swift 5.0
+	Running on macOS 10.15
+
+	Copyright 2019 The Tari Project
+
+	Redistribution and use in source and binary forms, with or
+	without modification, are permitted provided that the
+	following conditions are met:
+
+	1. Redistributions of source code must retain the above copyright notice,
+	this list of conditions and the following disclaimer.
+
+	2. Redistributions in binary form must reproduce the above
+	copyright notice, this list of conditions and the following disclaimer in the
+	documentation and/or other materials provided with the distribution.
+
+	3. Neither the name of the copyright holder nor the names of
+	its contributors may be used to endorse or promote products
+	derived from this software without specific prior written permission.
+
+	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+	CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+	OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+	DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+	CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+	SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+	NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+	HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+	OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var emojiContainer: UIView!
+    @IBOutlet weak var emojiLabel: UILabel!
+    @IBOutlet weak var copyEmojiButton: UIButton!
+    @IBOutlet weak var separatorView: UIView!
+    @IBOutlet weak var middleLabel: UILabel!
+    @IBOutlet weak var qrContainer: UIView!
+    @IBOutlet weak var qrImageView: UIImageView!
+
+    // MARK: - Override functions
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        customizeViews()
+    }
+
+    // MARK: - Private functions
+    private func customizeTitleLabel() {
+        let shareYourEmojiString = NSLocalizedString("Share your Emoji ID to receive Tari", comment: "profile title label")
+        let attributedString = NSMutableAttributedString(string: shareYourEmojiString,
+                                                         attributes: [.font: Theme.shared.fonts.profileTitleLightLabel ?? UIFont.systemFont(ofSize: 12.0),
+                                                                      .foregroundColor: Theme.shared.colors.profileTitleTextColor ?? .black,
```

Comment:
Also can safely force unwrap theme colors. Also a fallback hardcoded color will be an issue later when we implement dark mode